data/en/isnull.json

```diff
@@ -17,6 +17,14 @@
 	},
 	"links": [
 
+	],
+	"examples":[
+	{
+		"title":"isNull Example",
+		"description":"Returns true if the given object is null or the given expressions evaluates to null; Returns false is otherwise. ",
+		"code":"someObj = CreateObject(\"null\")\r\nisNull(someObj)",
```

Comment:
`CreateObject("null")` is not a valid way to produce a null, this example could simply be: `isNull( javaCast("null", "") )`